contracts/multihop/src/utils.rs

```diff
@@ -0,0 +1,15 @@
+use soroban_sdk::Vec;
+
+use crate::storage::Swap;
+
+pub fn verify_operations(operations: &Vec<Swap>) {
+    if operations.is_empty() {
+        panic!("Multihop: Operations empty");
+    }
+
+    for i in 0..operations.len() - 1 {
+        if operations.get(i).unwrap().ask_asset != operations.get(i + 1).unwrap().offer_asset {
```

Comment:
two questions I have:
is this the correct order
I can't use [vector.windows](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.windows) so that's why I used a simpler option, should I try and change it, or is it sufficient?

Reply:
added additional verification, the user might send us operation with just a single swap